README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
I would include an h1 heading above this link. Maybe have a #Tetris Game at the top of this file.